src/components/player/ShowAnswer.tsx

```diff
@@ -1,13 +1,14 @@
+import { state$ } from "../../state";
 
-function ShowAnswer( { answerCorrect }: {answerCorrect: boolean} ) {
+function ShowAnswer() {    
     
     // wait 5 sec
     // emit WS(waitingforquiz)
     // navigate to WaitingForNextQuestion
 
     return (
         <div>
-            {answerCorrect}
+            {state$.quiz.isAnswerCorrect.get() ? 'yay you got it right' : 'boo wrong'}
```

Comment:
Gordon added a show answer page from this PR. It's closed (not merged I think, should follow up on that)
https://github.com/yesh0907/Toohak-FE/pull/11